components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -202,6 +207,13 @@ bool BraveReferralsService::GetMatchingReferralHeaders(
 }
 
 void BraveReferralsService::OnFinalizationChecksTimerFired() {
```

Comment:
can this be called more than once? I don't think so, just want to double-check

Reply:
either way we may want to protect against calling PerformReferralAttestation more than once inside PrivateChannel